qpcr_analysis.py

```diff
@@ -263,7 +271,7 @@
         y = df_melting[df_melting.columns[index[i]]]
         plt.plot(x_melting,y)
         plt.ylim(-0.5,y_max)
-        ax.set_title(sample_names[index[i]], fontsize = 30)
+        ax.set_title(sample_names[index[i]], fontsize = 15)
         name_counter = name_counter + 1
         index_number += 1

```

Comment:
It's better to be consistent between `var = var + 1` and `var += 1`. The latter is, depending on the programming language, a bit more efficient.